daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2761,6 +2761,14 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 				return nil, errors.New("dm.thinp_autoextend_threshold must be greater than 0 and less than 100")
 			}
 			lvmSetupConfig.AutoExtendThreshold = per
+		case "dm.libdm_log_level":
```

Comment:
Somewhere in the comment it will be useful to put what values of libdm_log_level users can pass and what these map to.